packages/point-and-click/src/components/StepEditor.tsx

```diff
@@ -75,22 +79,19 @@ export const StepEditor: FC<{
   }, []);
 
   const currentSelector = useMemo(() => {
-    return step.trigger?.selector
-      ? step.trigger.selector
-      : (step.trigger?.path && toSelector(step.trigger.path)) || "";
+    return (
+      step.trigger?.selector ??
+      ((step.trigger?.path && toSelector(step.trigger.path)) || "")
```

Comment:
in that case we can use `??` instead of `||` since we used question marks in another part of the same expression for consistency

Reply:
Yes good point!!